### PR TITLE
Move content into a website

### PR DESCRIPTION
**Pull request by carolynvs**
This changes the repository from being read directly from GitHub using READMEs to putting the content into a statically generated site with Hugo. This repo is the secondary repo, the main website definition is in cncf/sig-contributor-strategy, and will be hosted at https://contribute.cncf.io.

* Move README's to _index.md, and move website content into hugo directory structure
* Add frontmatter to pages
* Integrate with Netlify. Deploys will trigger builds of the main site (sig-contributor-strategy) and previews will do a local build of the entire site on a separate netlify site just for previews.

I have a corresponding PR open to the other repo https://github.com/cncf/sig-contributor-strategy/pull/66. They should be merged basically together (first this then the one to the sig-repo).

### CODE REVIEW

**Comment by carolynvs** [comment]
@idvoretskyi This is ready to review/merge! 🎉

**Comment by idvoretskyi** [comment]
@carolynvs awesome, thanks!

I'll take a look.

**Comment by carolynvs** [comment]
Bumping in case this has fallen into the pit of github notifications. 😀

**Comment by idvoretskyi** [comment]
@carolynvs it has fallen actually, thanks for the reminder :)

File: README.md [hunk 1]
```diff
@@ -1,131 +1,14 @@
...
-=========================================
+# CNCF Contributors Website
 
-Welcome! Are you interested in contributing to one of CNCF hosted projects? This repository should help you. It contains information and guidelines about contributions to CNCF projects.
+Welcome! Are you interested in contributing to one of CNCF hosted projects? This
+repository contains information and guidelines about contributions to CNCF
+projects.
 
-CNCF offers multiple ways to start contributing to the CNCF ecosystem, including either foundation-wide and project-wide opportunities.
+<p align="center"><a href="https://contribute.cncf.io/contributors/">Contribution Guides to the CNCF Ecosystem</a></p>
```
Comment (idvoretskyi) [inline README.md:7]: This link points to https://github.com/cncf/contribute/graphs/contributors, we have to fix this a bit :)

**Review by idvoretskyi — COMMENTED** [review]
@carolynvs looks good, except for one URL that's occupied by the built-in GitHub functions.

Also, can you remind me where can we find the Netlify preview URL?

File: README.md (hunk 1 above)
Comment (carolynvs) [reply README.md:7]: Oops, yeah this assumed that we could merge the PR and quickly have the domain serve this new site. Let me change this to allow for some delay between merging and go-live

**Comment by jberkus** [comment]
Hey, we've merged the other parts of the website.  What's required to still merge this PR?

**Review by idvoretskyi — APPROVED** [review]
/lgtm

**Comment by idvoretskyi** [comment]
@carolynvs @jberkus here we go!

File: README.md (hunk 1 above)
Comment (idvoretskyi) [reply README.md:7]: @carolynvs we've merged this, please fix this.

**Pull request merged**